projects/graph/src/graph.py

```diff
@@ -105,3 +105,46 @@ def dft_recursion(self, starting_vertex_id, path = []):
         
         return path
 
+    def BFS(self, start, destination):
+         # Create an empty queue
+        q = Queue()
+        # Create an empty set of visited vertices
+        visited = []
+        # Put the starting vertex in our Queue
+        q.enqueue(start)
+        # While the queue is not empty
+        while q.size() > 0:
+            # Dequeue the first node from the queue
+            v = q.dequeue()
+            # If that node has not been visited...
+            if v not in visited:
+                # Mark it as visited
+                visited.append(v)
+                if v == destination:
+                    return visited
```

Comment:
While this will return the vertices that you had visited so far, per the specs, you want to return the (shortest) path from start to destination.  Therefore, you need a way to keep track of a path in addition to a list of vertices that you had visited.